deployment/charts/universalnode/templates/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app.kubernetes.io/app: {{ .Values.name }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/app: universalnode
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/app: universalnode
+    spec:
+      containers:
+      - name: universalnode
+        image: freeverseio/laos-universal-node:{{ .Values.imageTag }}
+        imagePullPolicy: IfNotPresent
+        ports:
+          - containerPort: 5001
+            name: rpc
+        resources:
+          requests:
+            memory: "512Mi"
+            cpu: "500m"
+          limits:
```

Comment:
I suggest changing limits to `2Gi` for `memory` and `2` for `cpu`. It's better to start with low requirements IMHO

Reply:
You've changed the memory in the requests and the CPU in the limits. I meant to change the values in the `limits` section only, sorry. Please change it to make it look like this:
```yml
requests:
  memory: "512Mi"
  cpu: "500m"
limits:
  memory: "2Gi"
  cpu: "2"
```